spec/factories/factories.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
```

Comment:
the name of this factory should not be `factories.rb`
please change it to `users.rb`